ui/components/multichain/product-tour-popover/product-tour-popover.scss

```diff
@@ -0,0 +1,32 @@
+.multichain-product-tour-menu {
+  width: 344px;
+  box-shadow: none;
+  left: -7px !important;
+  top: 10px !important; //important is required here since Menu has absolute position added via inline style in base component.
+
```

Comment:
This left position shifts the storybook component but it's added here in order to align it properly in the header. This component will be replaced with Popover as the base component once it's ready.